tests/test_core.py

```diff
@@ -71,9 +69,9 @@ def test_get_blocks_status_containing_tx(monkeypatch):
         {'id': 1}, {'id': 2}
     ]
     monkeypatch.setattr(
-        Bigchain, 'search_block_election_on_index', lambda x, y, z: blocks)
+        Bigchain, 'search_block_election_on_index', lambda x, y: blocks)
     monkeypatch.setattr(
-        Bigchain, 'block_election_status', lambda x, y: Bigchain.BLOCK_VALID)
+        Bigchain, 'block_election_status', lambda x, y, z: Bigchain.BLOCK_VALID)
```

Comment:
Well, we had to change it here because the method signature changed; if we had argument passthrough we wouldn't have had to. `x, y` isn't exactly the most useful method signature either.